site/docs/updating-bazel.md

```diff
@@ -5,11 +5,21 @@ title: Updating Bazel
 
 # Updating Bazel
 
-Bazel defined a [backwards compatibility policy](https://docs.bazel.build/versions/master/backward-compatibility.html)
+Bazel has a [backwards compatibility policy](https://docs.bazel.build/versions/master/backward-compatibility.html)
 (see [guidance for rolling out incompatible changes](https://www.bazel.build/breaking-changes-guide.html) if you are the author of one).
-This page summarized best practices on how to test and migrate your project with upcoming incompatible
+This page summarizes best practices on how to test and migrate your project with upcoming incompatible
 changes and how to provide feedback to the incompatible change authors.
 
+## Managing Bazel versions with Bazelisk
+
+Bazel team implemented a Bazel wrapper called [bazelisk](https://github.com/bazelbuild/bazelisk) that helps you
```

Comment:
Do we normally use  |bazel team| or |the bazel team|

Reply:
Done.